Resources/Prototypes/Entities/Mobs/NPCs/animals.yml

```diff
@@ -1457,6 +1457,67 @@
       description: ghost-role-information-giant-spider-description
     - type: GhostTakeoverAvailable
 
+- type: entity
+  name: clown spider
+  parent: MobGiantSpider
+  id: MobClownSpider
+  description: Combines the two most terrifying things in existence, spiders and clowns.
+  components:
+    - type: Sprite
+      drawdepth: Mobs
+      layers:
+      - map: ["enum.DamageStateVisualLayers.Base"]
+        state: clown
+        sprite: Mobs/Animals/clownspider.rsi
+    - type: Faction
+      factions:
+        - Xeno
+    - type: InputMover
+    - type: MobMover
+    - type: HTN
+      rootTask: SimpleHostileCompound
+    - type: Butcherable
+      spawned:
+        - id: MaterialBananium1
+          amount: 1
+    - type: GhostRole
+      makeSentient: true
+      name: ghost-role-information-giant-spider-name
+      description: ghost-role-information-giant-spider-description
+    - type: GhostTakeoverAvailable
+    - type: DamageStateVisuals
+      states:
+        Alive:
+          Base: clown
+        Critical:
+          Base: dead
+        Dead:
+          Base: dead
+    - type: MobThresholds
+      thresholds:
+        0: Alive
+        100: Critical
```

Comment:
Just remove crit state and have dead only.

Reply:
removed crit state.